src/pages/resultsView/mutation/ResultsViewMutationTable.tsx

```diff
@@ -116,4 +145,50 @@ export default class ResultsViewMutationTable extends MutationTable<IResultsView
         this._columns[MutationTableColumnType.EXON].headerRender = 
             () => <span style = {{display:'inline-block'}}>Exon<br/>({this.props.totalNumberOfExons} in total)</span>;
     }
+
+    @computed private get hasCcfMCopies():boolean {
```

Comment:
It might be better combing the following three methods together. They seem to do similar work and have the same logic.

Reply:
@zhx828 can you clarify? Since the three functions are checking for different members, are you proposing a single function that checks for a field passed in as an arg?  e.g `missingRequiredField(fieldName)`

I tried that and it looks like I can't use a `get` function with args. But if I remove the `get` the frontend throws and error since `@computed` only works with getter functions. Is the `@computed` tag necessary?

Reply:
@averyniceday please take a look the two comments I just added at the bottom of the page.

Reply:
And here you are dealing with pure util function. No computed or getter needed.